torch/_inductor/kernel/flex_decoding.py

```diff
@@ -134,18 +138,13 @@ def flex_decoding_grid(batch_size, num_heads, n_keys, d_model, meta):
         order=(1, 0)
     )
     if SAFE_M_BOUNDARY:
-        q = tl.load(Q_block_ptr)
+        q = tl.load(Q + q_offset + q_range)
```

Comment:
It looks like you are no longer using the Q block ptr?

Reply:
It is easier because we're loading a group of query head x query len x head dim. Is there any benefit for using Q block ptr ?

Reply:
Ohh I was saying you can probably remove the Q_block_ptr creation above